lib/digest.js

```diff
@@ -1,21 +1,34 @@
-var md5 = require('md5')
-var _ = require('lodash')
+const md5 = require('md5')
+const _ = require('lodash')
 
 function key(axiosConfig) {
   //Content-Length is calculated automatically by Axios before sending a request
   //We don't want to include it here because it could be changed by axios
 
+  let url = axiosConfig.url
+  let method = axiosConfig.method
+  let data = axiosConfig.data
+  let headers = axiosConfig.headers
 
-  var baseConfig = {
-    url: axiosConfig.url,
-    method: axiosConfig.method,
-    data: axiosConfig.data,
-    headers: _.omit(axiosConfig.headers, [
-      'Content-Length', 'content-length'
-    ])
+  if (_.isString(data)) {
+    data = JSON.parse(data)
```

Comment:
there was a mis-match b/c the request was an object but the response was JSONified